### PR TITLE
Bug #5973 ubs admin tariffs receiving station

### PR DESCRIPTION
**Pull request by 2or5**
## Summary of issue

The message about mandatory filling in the "receiving station" field is not displayed when only the location and the courier tags are selected when creating a new pricing card #5973

## Summary of change

Add validation for receivingStation

### CODE REVIEW

File: service-api/src/main/java/greencity/dto/AddNewTariffDto.java [hunk 1]
```diff
@@ -26,5 +26,6 @@ public class AddNewTariffDto {
     private Long courierId;
     @NotEmpty
     private List<@Min(1) Long> locationIdList;
+    @NotEmpty
```
Comment (RomaMocherniuk) [inline service-api/src/main/java/greencity/dto/AddNewTariffDto.java:29]: please add test if `receivingStationsIdList` is empty

File: service-api/src/test/java/greencity/dto/courier/tariff/AddNewTariffDtoTest.java [hunk 2]
```diff
@@ -47,6 +48,20 @@ void addNewTariffDtoWithValidNullFieldsTest() {
...
+        var dto = ModelUtils.getAddNewTariffWithNullFieldsDto();
+        dto.setReceivingStationsIdList(Collections.emptyList());
+
+        ValidatorFactory factory = Validation.buildDefaultValidatorFactory();
+        final Validator validator = factory.getValidator();
+
+        Set<ConstraintViolation<AddNewTariffDto>> constraintViolations =
+            validator.validate(dto);
+
+        assertThat(constraintViolations).hasSize(1);
```
Comment (RomaMocherniuk) [inline service-api/src/test/java/greencity/dto/courier/tariff/AddNewTariffDtoTest.java]: Can we check the validation error too?
